Source/Scene/ModelExperimental/PntsLoader.js

```diff
@@ -206,7 +208,7 @@ function decodeDraco(loader, context) {
     .catch(function (error) {
       loader.unload();
       loader._state = ResourceLoaderState.FAILED;
-      const errorMessage = "Failed to load Draco";
+      const errorMessage = "Failed to load pnts";
```

Comment:
ah I thought it was a copy/paste error but I didn't read the function name. I'm going to change it to `Failed to load Draco pnts` to be more specific, since I found "Failed to load Draco" confusing (not specific enough to know which file to look in)